src/notation/view/noteinputcursor.cpp

```diff
@@ -48,7 +48,7 @@ void NoteInputCursor::paint(mu::draw::Painter* painter)
         const StaffType* staffType = state.staff ? state.staff->staffType() : nullptr;
 
         if (staffType) {
-            staffType->drawInputStringMarks(painter, state.currentString, state.currentVoiceIndex, cursorRect);
+            staffType->drawInputStringMarks(painter, state.currentString, static_cast<int>(state.currentVoiceIndex), cursorRect);
```

Comment:
I'd rather update the signature of `StaffType::drawInputStringMarks` to take `voice_idx_t` instead of `int`.

Reply:
True, and in this case esp. easy, as it is used in only one place

Reply:
Done